.golangci.yml

```diff
@@ -53,6 +53,45 @@ issues:
   max-same-issues: 10000
 
 linters-settings:
+  gosec:
+    # To select a subset of rules to run.
+    # Available rules: https://github.com/securego/gosec#available-rules
+    # Default: [] - means include all rules
+    includes:
+      #  - G101 # Look for hard coded credentials
```

Comment:
Thanks! but please don't add more in this PR :D

Reply:
It's fine to leave it here, I just wanted to make sure you don't re-add the rest here.